packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -316,10 +318,55 @@ export function createEditPageNode({
                                                 return <Typography>/</Typography>;
                                             }
 
+                                            function createCompletePath(slug: string) {
+                                                return (
+                                                    <>
+                                                        {parentPath ?? ""}
```

Comment:
```suggestion
                                                        {parentPath}
```

I believe this is not necessary, as undefined or null isn't rendered anyway. Correct me if I'm wrong 😁

Reply:
I also remove this function as it's not necessary anymore